source/parser.hera

```diff
@@ -3994,6 +3994,11 @@ TypeParameters
   __ "<" TypeParameter+ __ ">" ->
     return { ts: true, children: $0 }
 
+# TypeParameters without the leading whitespace
+CompactTypeParameters
```

Comment:
I think this should be `CompactTypeArguments` and `TypeArgument+` since it is part of the application.

Parameters for declaration, arguments for application.